app/src/test/kotlin/fr/free/nrw/commons/explore/search/SearchActivityUnitTests.kt

```diff
@@ -81,11 +76,14 @@ class SearchActivityUnitTests {
     @Mock
     private lateinit var searchCategoryFragment: SearchCategoryFragment
 
+    private lateinit var binding : ActivitySearchBinding
+
     @Before
     fun setUp() {
         MockitoAnnotations.openMocks(this)
         activity = Robolectric.buildActivity(SearchActivity::class.java).create().get()
         context = ApplicationProvider.getApplicationContext()
+        binding = ActivitySearchBinding.inflate(activity.layoutInflater)
```

Comment:
Looks like you can drop the `@Mock` annotation on the context (since you're giving a real `Context` here).  Also, the _binding_ appears to be unused in the rest of the test and could be deleted.